Packs/Genians/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Genians",
+    "description": "Use the Genians integration to block IP using assign tag and unassign tag.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Genians",
+    "url": "https://www.genians.com/",
```

Comment:
@genians-rnd is this a valid url to get support from?

Reply:
The url is https://docs.genians.com/release/en/
I think the docs page would be nice.

Reply:
Can I request 'pull requests' again..?

Reply:
No need for a new pr, just submit the needed change

Reply:
I changed it.